rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/notifications/OpenUnityProjectAsFolderNotification.kt

```diff
@@ -49,6 +57,9 @@ class OpenUnityProjectAsFolderNotification(private val project: Project, private
                     ProjectUtil.closeAndDispose(project)
                     WelcomeFrame.showIfNoProjectOpened()
                 }
+                if (hyperlinkEvent.description == "reopen") {
+                    StartUnityAction.StartUnity(project, "-executeMethod", "JetBrains.Rider.Unity.Editor.RiderMenu.MenuOpenProject")
```

Comment:
I think move `-executeMethod` and static method arguments into the `StartUnity` method. Then `StartUnity` is the only place that has to know about Unity's command line args. Perhaps even rename to something like `OpenUnityProject` or `OpenProjectInUnity`. If we need to reuse this in some other place with other parameters or static method names, we can refactor at that time.

Reply:
What happens if the plugin isn't installed? Is there an error?

Reply:
Should this also close the current project?

Reply:
Plugin is expected to be automatically installed by the same rules, which show this notification.